CHANGELOG.md

```diff
@@ -24,6 +24,20 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## ❗ BREAKING ❗
 ## 🚀 Features
 
+- **Container images are now available** ([PR #764](https://github.com/apollographql/router/pull/764))
+
+  We now build container images More details at:
+    https://github.com/apollographql/router/pkgs/container/router
+
+  You can use the images with docker, for example, as follows:
+    e.g.: docker pull ghcr.io/apollographql/router:v0.1.0-preview.1
```

Comment:
I'm not a big fan of the "latest" tag. This blog post sums up most of my views (although I probably don't hold them as strongly as the author): https://vsupalov.com/docker-latest-tag/

kubernetes notes recommend avoiding using "latest". https://kubernetes.io/docs/concepts/containers/images/#:~:text=Note%3A,42.0%20.

Anyway: I don't feel it very strongly, but I do think that "latest" is not a great pattern to adopt.

That's just supposed to be an example to show people what to pull. We could just update it to something like:
```
docker pull ghcr.io/apollographql/router:<image version>
```
but I suppose that's less literally an example.

Reply:
Ahh, sorry, this is the changelog which is fine. But in the main docs I think we should either specify the `<image version>` or `latest`.

I don't think what we put here can be constituted as run with this in production advice, and it will cause users to download old versions of the router as their first experience of using the thing.

Reply:
I've replace "v0.1.0-preview.1" with "<image version>". In an explanatory note I illustrate how to replace <image version> with the user's chosen version to make it clear that "<image version>" isn't actually a choice. For illustrative purposes, I'm using "v0.1.0-preview.1" for this purpose, since it's the only release we have images for right now.

Note: It would be great if there was a way to update this in the docs when we are cutting a release, but not this issue.

Reply:
done: 8de780de6e126438e0f4ba4287ac63ffaff55a37